tests/pytest_framework/functional_tests/logpath/test_flags_catch_all.py

```diff
@@ -66,7 +61,7 @@ def test_flags_catch_all(tc):
     # message should arrived into destination1
     assert tc.format_as_bsd(expected_bsd_message) in dest1_logs
 
-    dest2_logs = file_destination2.read_log()
+    dest2_logs = catch_all_destination.read_log()
```

Comment:
Might not be the scope of this change, but `dest2_logs` is not so talkative. IMO `catch_all_dest_logs` or something might fit better. Same applies for `dest1_logs`

Reply:
Fixed thanks.